src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/LegacyPbaResponse.java

```diff
@@ -19,7 +19,8 @@ public LegacyPbaResponse(List<String> payment_accounts) {
     @SuppressWarnings({"checkstyle:ParameterName"})
     private List<String> getPaymentAccounts(List<String> payment_accounts) {
 
-        return this.payment_accounts = payment_accounts.stream().map(payment_account ->
+        payment_accounts.stream().map(payment_account ->
                 payment_account).collect(Collectors.toList());
+        return this.payment_accounts;

```

Comment:
Not what you want to do. This is removing the functionality that was previously there with the assignment. Please revert the code. Or else this will lead to a production issue.